README.md

```diff
@@ -32,11 +32,24 @@ Here are some useful commands:
 
 ## Publishing a new version of a package
 
-If this is the first time the package is published to npm, running `yarn lerna publish --no-private` [doesn't seem to work](https://github.com/lerna/lerna/issues/1821). Instead, you should `cd` into your package and run `yarn publish --access public`. `yarn` will then ask the new version the package should have.
-
-If the package has already been published on npm, you can use `yarn lerna publish --no-private` in the root directory of the project.
-
-For each of these two tasks, you will need to have an account on [npmjs.com](https://www.npmjs.com/) and to be added to the [@bam.tech](https://www.npmjs.com/settings/bam.tech/packages) organization.
+The publication is done automatically by this [Github Workflow](.github/workflows/publish.yml) when a new tag is pushed to the repository.
+All you have to do is the versioning of the packages you want to publish.
+
+> You need to be on the main branch and have the repo write access to publish a new version.
+
+Run `yarn lerna version` to start the process. It will ask you which packages you want to publish and which version you want to publish them under.
```

Comment:
I would have explained a bit more what this does, so bump version in package json, push commit, create tag and push tag